src/components/Herosection/hero.css

```diff
@@ -0,0 +1,53 @@
+.heroContainer {
+  width: 100%;
+  position: relative;
+  height: calc(100vh-80px);
+  /* background-color: blue; */
+}
+
+.heroBackgroundImgContainer {
+  width: 100%;
+  height: 90vh;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url("./bgImage.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  object-fit: cover;
+  /* opacity: ; */
+}
+
+.heroTaglineContainer {
+  width: 50%;
+  padding: 0 2.1875rem;
```

Comment:
You may want to decare this padding property in one line. e.g,  padding:  3rem 2.5rem 2rem;
The first value is for the top, the second value is for both right and left and the last value is for the bottom.
Read about shorthand for padding.